src/main/kotlin/lotto/enums/LottoRank.kt

```diff
@@ -11,10 +11,18 @@ enum class LottoRank(
     SIX_COLLECT(6, 2_000_000_000),
     NON_COLLECT(0, 0);
 
-
     companion object {
         fun getLottoRankByCount(count: Int): LottoRank {
             return values().find { it.count == count } ?: NON_COLLECT
         }
+
+        fun getLottoPrizeMoneyByCount(collectLottoRanks: List<LottoRank>, collectBonusCount: Int): Int {
+            var winningMoney = 0
+            collectLottoRanks.forEach { rank ->
+                winningMoney += rank.prizeMoney
+            }
```

Comment:
```suggestion
            var winningMoney = collectLottoRanks.sumOf { rank ->
                rank.prizeMoney
            }
```
sumOf를 활용하시면 변수 선언과 동시에 초기화가 가능합니다. 😄